src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -128,14 +128,17 @@ export default class AircraftCommander {
             const r_log = _map(response, (r) => r.log).join(', ');
             const r_say = _map(response, (r) => r.say).join(', ');
 
-            UiController.ui_log(`${aircraft.callsign}, ${r_log} ${response_end}`, redResponse);
-            speech_say(
-                [
-                    { type: 'callsign', content: aircraft },
-                    { type: 'text', content: `${r_say} ${response_end}` }
-                ],
-                aircraft.pilotVoice
-            );
+            // no need for reading back or writing to the UI for preSpawn commands
+            if (!isPreSpawn) {
+                UiController.ui_log(`${aircraft.callsign}, ${r_log} ${response_end}`, redResponse);
+                speech_say(
+                    [
+                        { type: 'callsign', content: aircraft },
+                        { type: 'text', content: `${r_say} ${response_end}` }
+                    ],
+                    aircraft.pilotVoice
+                );
+            }
```

Comment:
As you know, we skip this when we're prespawning, because we don't display the response. Rather than wrapping this in an if statement, since there's nothing else after this section, we should instead return early before it.

In fact, we can also skip some stuff above this, so let's do that! After the second for loop, at line 116, we can add `if (preSpawn) { return; }`, and then this section we can just continue inline without the if.

Reply:
done